how-to/misc/using-the-epa-orchestrator.rst

```diff
@@ -0,0 +1,280 @@
+Using the EPA Orchestrator
```

Comment:
Our [style guide](https://docs.ubuntu.com/styleguide/en/) recommends using imperatives - “Imperatives should be used for 'How to...' style docs instead of gerunds (i.e. "Use the EPA Orchestrator" rather than "Using the EPA Orchestrator")
The use of imperative delivers a clear command and helps users understand they need to perform an action. The use of gerund can hint at a description, and can lead to confusion, as users might think an explanation will follow.